modules/menu/css/README.md

```diff
@@ -40,6 +40,20 @@ Use `.wdc-menu` to create a menu.
 </div>
 ```
 
+### Variants
+
+#### Grow
```

Comment:
I snuck in support for the grow prop in the React Menu since it was missing from the previous refactor(Menu) PR. Let me know if this should be handled as a separate PR.

Reply:
Good catch! Fine to include it here.